wordpress-mobile/WordPress-iOS
# Issue/8045 aztec image links

Fixes #8045 - Add links to images in posts from its details editor.

Note that the Podfile points to the corresponding version of WordPress-Aztec-iOS that makes linked images possible:
`pod 'WordPress-Aztec-iOS', :git => 'https://github.com/wordpress-mobile/AztecEditor-iOS.git', :commit => '3130fee7bcea8ab7c246be340ecc42a991de4576'`
... which will obviously need to be cleaned up later on.

To test:

1. run the WordPress-iOS app and make sure you have it set to use the "Visual 2.0" editor
2. tap the appropriate button to begin creating a new post
3. tap the text area labeled "Share your story here..."
4. tap the "+" button in the lower left corner to add an image
5. once you've added the image, tap on it to show "Tap for options"
6. tap the image again, then tap "Details"
7. tap "Link To" and enter a URL
8. tap "Media Settings" to go back, and note that the URL is now shown
9. tap "Done"
10. in the format bar at the bottom, tap the `</>` button to show that the link URL is shown within the HTML (with an `<a href></a>` containing the URL around the img tag)
11. if you want, edit the URL within the HTML
12. tap the `</>` button again to show that the HTML is properly converted back to an NSAttributedString... and that the image displays properly
13. tap the image and bring up the media details again, and observe that the link URL is still set (because it was properly converted back from HTML)

Needs review: @diegoreymendez
